src/main/java/Deadline.java

```diff
@@ -0,0 +1,29 @@
+public class Deadline extends Task{
+    protected String by;
+    protected String originalInput;
+    public Deadline(String description, String by, String originalInput) {
```

Comment:
To improve code readability, you may wish to leave a line spacing the code block of class attributes and the constructor (`Deadline()`) here.